packages/components/src/Tooltip/Tooltip.tsx

```diff
@@ -147,17 +157,3 @@ export function Tooltip({
     }, []);
   }
 }
-
-interface TooltipPortalProps {
-  readonly children: ReactNode;
-}
-
-function TooltipPortal({ children }: TooltipPortalProps) {
-  const mounted = useIsMounted();
-
-  if (!mounted?.current) {
-    return null;
-  }
-
-  return ReactDOM.createPortal(children, document.body);
-}
```

Comment:
I didn't have to get rid of this. it's not doing any harm to have it be SSR safe. we just don't explicitly need it anymore.

Reply:
decided to keep it. it wasn't hurting anyone.